backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/VerkuendungImportProcess.java

```diff
@@ -29,7 +26,7 @@ public class VerkuendungImportProcess {
   private Instant finishedAt;
 
   @Default
-  private List<VerkuendungImportProcessDetail> detail = new ArrayList<>();
+  private List<VerkuendungImportProcessDetail> detail = List.of();
```

Comment:
just to be sure: you want this list to be immutable?

Reply:
I don't think it should be immutable..